scripts/proteinInteractionMINT/README.md

```diff
@@ -0,0 +1,63 @@
+# Scripts for importing ontology dataset from the Molecular INTeraction database (MINT)
```

Comment:
Can we move MINT to another PR? This is large enough as such.

Reply:
Sure I will move MINT to another PR